src/mame/netronics/exp85.cpp

```diff
@@ -41,24 +54,33 @@
 #include "imagedev/cassette.h"
 #include "sound/spkrdev.h"
 
+#define VERBOSE 0 //(1U)
+#include "logmacro.h"
 
 namespace {
 
-#define I8085A_TAG      "u100"
-#define I8155_TAG       "u106"
-#define I8355_TAG       "u105"
+#define I8085A_TAG       "u100"
```

Comment:
A lot of the devs (myself included) have largely moved away from #defining all of the tag names because it doesn't often buy you much.  But we don't prohibit it either, so if you like it we won't stop you.

Reply:
I took your advice and removed these defines; the names "uNNN" reflect the id of the component on the circuit diagrams published by netronics.